test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -69,6 +72,28 @@ function teardown() { _default_teardown ; }
   _should_receive_spam_at '/var/mail/localhost.localdomain/user1/.Junk/new/'
 }
 
+@test "(enabled + MARK_SPAM_AS_READ=1) should mark spam message as read" {
+  export CONTAINER_NAME=${CONTAINER2_NAME}
```

Comment:
Adding a comment for context to maintainers/reader of test + use a new container name:

```suggestion
# NOTE: Same as test for `CONTAINER3_NAME`, only differing by ENV `MARK_SPAM_AS_READ=1` + `_should_receive_spam_at` location
@test "(enabled + MARK_SPAM_AS_READ=1) should mark spam message as read" {
  export CONTAINER_NAME=${CONTAINER4_NAME}
```

At the top of the test file, add your new container name after `CONTAINER3_NAME`:

```bash
CONTAINER4_NAME='dms-test_spam-amavis_env-mark-spam-as-read-1'
```